frontend/src/app/utils/form-helpers/task-form.helper.ts

```diff
@@ -0,0 +1,67 @@
+import { endOfDay, startOfDay } from 'date-fns';
```

Comment:
I guess appropriate place for this is in exact form component where it should be used - task-form. To avoid usage from other components.
I guess it is valid for other helpers in utils. E.g. timeline helper. It has high coupling for your timeline component and it does not seem that it could be used somewhere else. So it should not be in generic utils folder. Review other utils for same issue.
here is a short reminder of what coupling is: https://medium.com/german-gorelkin/low-coupling-high-cohesion-d36369fb1be9

Reply:
Fixed